src/models/data.cpp

```diff
@@ -272,6 +272,27 @@ QString prettifySymbol(const QStringRef& str)
 
     return result;
 }
+
+void mergeBottomUpResults(const BottomUp& a, const BottomUp* b, BottomUp* result_node, const Costs& costs_a,
```

Comment:
please rename to `diffBottomUpResults` - merging is used in the code base to indicate that we are merging by symbol e.g. which this isn't doing